client/src/components/productDetails/OverviewMain.jsx

```diff
@@ -251,7 +254,7 @@ console.log(response.data.results)
           <Carousel fullSizePhotos={this.state.fullSizePhotos} smallSizePhotos={this.state.smallSizePhotos} currentMainThumbnail={this.state.currentMainThumbnail} />
 
           <div className='containerChild2'>
-            <ProductInformation productInfo={this.state.productInfo} allResultsArray={this.state.allResultsArray} currentStyleIndex={this.state.currentStyleIndex} avgRating = {this.props.avgRating}/>
+            <ProductInformation productInfo={this.state.productInfo} allResultsArray={this.state.allResultsArray} currentStyleIndex={this.state.currentStyleIndex} avgRating = {this.props.avgRating} updateProductName = {this.props.getProductName}/>
```

Comment:
Add each props function on a new line for readability

<ProductInformation 
  productInfo={this.state.productInfo} 
  allResultsArray={this.state.allResultsArray} 
  currentStyleIndex={this.state.currentStyleIndex} 
  avgRating = {this.props.avgRating} 
  updateProductName = {this.props.getProductName}
/>